rpg-sheet/src/components/InventoryItems/InventoryItems.js

```diff
@@ -0,0 +1,14 @@
+export const InventoryItems = (item) => {
```

Comment:
```suggestion
export const InventoryItems = ({ item }) => {
```

Cuidado com o uso das props aqui!